e2e/tshark/tshark_test.go

```diff
@@ -0,0 +1,69 @@
+package basic
+
+import (
+	"context"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/celestiaorg/knuu/pkg/knuu"
+)
+
+func TestTshark(t *testing.T) {
+	t.Parallel()
+	// Setup
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	knuu, err := knuu.New(ctx)
+	require.NoError(t, err, "Error creating knuu")
+
+	scope := knuu.Scope()
+	t.Log(scope)
+
+	instance, err := knuu.NewInstance("alpine")
+	require.NoError(t, err, "Error creating instance")
+
+	err = instance.SetImage(ctx, "docker.io/alpine:latest")
+	require.NoError(t, err, "Error setting image")
+
+	err = instance.SetCommand("sleep", "infinity")
+	require.NoError(t, err, "Error setting command")
+
+	err = instance.EnableTsharkCollector(
+		"10Gi", // Example volume size
+		os.Getenv("S3_ACCESS_KEY"),
+		os.Getenv("S3_SECRET_KEY"),
+		os.Getenv("S3_REGION"),
+		os.Getenv("S3_BUCKET_NAME"),
+		"tshark/"+scope,
+	)
+	require.NoError(t, err, "Error enabling tshark collector")
+
+	err = instance.Commit()
+	require.NoError(t, err, "Error committing instance")
+
+	t.Cleanup(func() {
+		require.NoError(t, instance.Destroy(context.Background()))
+	})
+
+	// Test logic
+
+	err = instance.Start(ctx)
+	require.NoError(t, err, "Error starting instance")
+
+	err = instance.WaitInstanceIsRunning(ctx)
+	require.NoError(t, err, "Error waiting for instance to be running")
+
+	wget, err := instance.ExecuteCommand(ctx, "echo", "Hello World!")
+	require.NoError(t, err, "Error executing command")
+
+	// wait for 1 minute to upload network traces to s3
+	time.Sleep(1 * time.Minute)
+
+	assert.Contains(t, wget, "Hello World!")
```

Comment:
It isn't clear why you need this sleep? 
If this is purely for debugging purposes, I'd comment it out and update the comment to specifically explain using it for debugging. 
The assertion on `wget` should be deterministic from line 62 and not impacted by a sleep after the command is executed.

Reply:
This test checks if the `tshark` data is successfully uploaded to s3. This I have done manually.
So in theory we should check if a file was created at the expected path and is updated regularly.

Reply:
If there is a way to programmatically check for the file, we should do that in the test. 

If there isn't (as of today) we should turn this into a follow up and figure out how to do it. I'd also add a TODO comment here in the test that we need to check the file so that is clear.

Either way the sleep should be removed.